neo4j/conf.py

```diff
@@ -256,6 +263,12 @@ class WorkspaceConfig(Config):
     """ WorkSpace configuration.
     """
 
+    #: Session Connection Timeout
+    session_connection_timeout = 120.0  # seconds
```

Comment:
Shouldn't it be inf?

Reply:
I 100%... nah inf% should be! :open_mouth: